docs/recipes/precompiling-with-webpack.md

```diff
@@ -0,0 +1,25 @@
+In the [README](https://github.com/avajs/ava#transpiling-imported-modules) it mentions precompiling your imported modules as an alternative to JIT compilation, but it doesn't explain how. This recipe shows how to do this using webpack.
+
+webpack.config.js
+```js
+var nodeExternals = require('webpack-node-externals');
+
+module.exports = {
+  entry: ['./src/tests.js'],
+  target: 'node',
+  output: {
+    path: '_build',
+    filename: 'tests.js'
+  },
+  externals: nodeModules,
```

Comment:
I have not tested the recipe. But we should instead declare `nodeExternals` instead of `nodeModules`? I have a little difficulty understanding the logic. If what you wrote is correct then it might be good to explain why we should do this.

What do you think?